schemas/credentials.json

```diff
@@ -451,6 +451,33 @@
                 "encrypt": true,
                 "sectionEnd": "Monit Alerts Email Server",
                 "propertyOrder": 470
+            },
+            "nuh_notification_app_1_username": {
+                "type": "string",
+                "title": "NUH notification application 1 username",
+                "description": "Username for NUH notification application",
+                "encrypt": true,
+                "sectionBegin": "NUH notification application",
+                "propertyOrder": 480
+            },"nuh_notification_app_1_password": {
```

Comment:
Please add new-lines before the property name to be consistent with other fields.

Reply:
done